src/components/common/Sidebar/Sidebar.js

```diff
@@ -34,7 +34,7 @@ const Sidebar = ({ authService, userProfile }) => {
   const [render, updateRender] = useState(1);
   const { authState } = useOktaAuth();
 
-  const toggleTheme = useTheme();
+  const [theme, toggleTheme] = useTheme();
```

Comment:
Please remove 'ContainerOutlined' as it's warning us in our linter.

Reply:
Resolved in latest commit below